src/app/business/block/file-share-detail/file-share-detail.component.html

```diff
@@ -178,30 +178,20 @@ <h3>{{i18n.keyID['sds_block_volume_base_info']}}</h3>
                             </form-item>
                             <form-item [required]="true" label="{{label.user}}">
                                 <div class="ui-grid-row volume-item" *ngFor="let AclsItem of aclsItems;index as i">
-                                    <!-- <p-dropdown [options]="availabilityip" formControlName="{{'user'+i}}" [style]="{'min-width':96}"></p-dropdown> -->
                                     <div class="ui-grid-col-9" style="margin-bottom: -10px">
-                                        <input type="text" pInputText formControlName="{{'userInput'+i}}" [style]="{'width':150}" />
-                                        <div class="ui-message-container" *ngIf="createAclsFormGroup.controls['userInput'+i] && !createAclsFormGroup.controls['userInput'+i].valid&&createAclsFormGroup.controls['userInput'+i].dirty">
+                                        <input type="text" pInputText formControlName="{{'userInput'+i}}" [style]="{'width':150}" (keyup)="userIpChnage(i)"/>
```

Comment:
fix typo userIpChnage() -> userIpChange()

Reply:
I have just modified it. Please check it

Reply:
I still see that the function name is not changed.  
(keyup)="userIpChnage(i)" -> (keyup)="userIpChange(i)"